apps/arena/lib/arena/game_launcher.ex

```diff
@@ -68,15 +66,16 @@ defmodule Arena.GameLauncher do
     Process.send_after(self(), :launch_game?, 300)
     diff = System.monotonic_time(:millisecond) - state.batch_start_at
 
-    if length(clients) >= @clients_needed or (diff >= @start_timeout_ms and length(clients) > 0) do
+    if length(clients) >= Application.get_env(:arena, :clients_needed_in_match) or
```

Comment:
Players instead of clients seems more accurate imo.
```suggestion
    if length(clients) >= Application.get_env(:arena, :players_needed_in_match) or
```